src/clients/apiClient.ts

```diff
@@ -22,7 +22,7 @@ export const checkBackendConnection = async (): Promise<boolean> => {
 }
 
 export async function createSighting(newSighting: NewSighting): Promise<Response> {
-    const response = await fetch(`https://${backendUrl}/createsighting`, {
+    const response = await fetch(`https://${backendUrl}/sightings/submit`, {
```

Comment:
FYI this is the backend path, so should be `controller route/endpoint route` which could be different to the frontend path - this is fine for now

Reply:
Noted, thank you :)